dev-pip-requirements

```diff
@@ -0,0 +1,15 @@
+-r pip-requirements
+-r doc-pip-requirements
+Cython
+jinja2
+pyyaml
+scipy
+pandas
+h5py
+bs4
```

Comment:
I think you want `beautifulsoup4` here instead. `bs4` gets you to here: https://pypi.python.org/pypi/bs4/